server/compactor/src/test/java/org/apache/accumulo/compactor/CompactorTest.java

```diff
@@ -107,6 +108,11 @@ public AtomicReference<FileCompactor> getFileCompactor() {
       return new AtomicReference<>(compactor);
     }
 
+    @Override
+    public Duration getCompactionAge() {
```

Comment:
`toNanos` is called on the return value in almost all cases. I'm wondering if you want to change this to `Duration.ZERO` to prevent a `NullPointerException`. Same comment for the same change in `ExternalDoNothingCompactor`.

Reply:
Yea good idea. This was causing the test failure but is fixed as of 12dc0f6.